client/me/connected-applications/index.jsx

```diff
@@ -97,22 +121,23 @@ const ConnectedApplications = React.createClass( {
 	},
 
 	renderConnectedApps: function() {
-		return this.props.connectedAppsData.initialized
+		return ( this.props.connectedAppsData.initialized
```

Comment:
Nit: indentation in this section is wonky 😕

Reply:
Yeah, to be honest, I'm not a big fan either. I think our time will be better spent leaving these styling issues to `prettier` in the future.